### DISCUSSION TITLE
Integration Not showing up when searching

### DISCUSSION THREAD

**Discussion by chilump** [sbidy/wiz_light#205] [General]
Hi,
I have Wiz app and lights installed and working on my network. When I try to search for WIZ integration within HACS it doesn't show for me. I look within the custom section and I also don't see it there when I search. Thoughts on what I may be doing wrong.

**Comment by ksignorini** [comment]
I added the repository using the ... menu in HACS and I can get the integration installed in HACS. Three vertical dots.

**Reply by chilump** [reply]
When I select the Custom Repository option, do I need to type in a path and select the repository option or should it load when I type in wiz and select repository?
Thanks for all your work on this!

**Reply by chilump** [reply]
I figured it out, had to type in https://github.com/sbidy/wiz_light path. 
Thanks!!!
